### PR TITLE
Integration test fix

### PR DESCRIPTION
**Pull request by tmatsuo**
This needs `google/cloud` 0.33 release

### CODE REVIEW

File: testapps/integration/web/index.php [hunk 1]
```diff
@@ -29,12 +29,12 @@
 });
 
 $app->post('/logging_standard', function (Request $request) {
-    $logName = $request->get('log_name');
-    $token = $request->get('token');
+    $logName = $request->request->get('log_name');
```
Comment (chingor13) [inline testapps/integration/web/index.php]: It looks like this isn't provided to the logging_standard test anymore, but it is to the logging_custom test.

Comment (tmatsuo) [inline testapps/integration/web/index.php]: Yeah the README says so, but the code actually adds log_name field.
https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/integration_tests/testsuite/test_logging_standard.py#L35
and
https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/integration_tests/testsuite/test_util.py#L74

Adding @nkubala to confirm

Comment (nkubala) [inline testapps/integration/web/index.php]: Ahh, sorry this is confusing. The `logging_standard` test doesn't make use of this field at all, even though it is actually provided by the client. I just combined the payloads of the two tests to simplify client code a bit. The standard logging tests should just be using the language's standard logging module to write the log entry, without specifying a name. Not all languages have this feature (not sure if PHP does; in python this is accomplished by simply doing `logging.info(token)`, for level `INFO`); if PHP doesn't have this, feel free to opt out of this test.

Comment (chingor13) [inline testapps/integration/web/index.php]: Ok, it looks like this code can just be moved to the logging_custom test. Our logger doesn't look like it has a default log name"

Comment (nkubala) [inline testapps/integration/web/index.php]: Ok, yeah sorry I should have been a bit more explicit

Comment (tmatsuo) [inline testapps/integration/web/index.php]: I got it. Then it is worth testing logs on stderr. I moved the current test to logging_custom and use stderr in logging_standard. @chingor13 PTAL

File: testapps/integration/web/index.php [hunk 2]
```diff
@@ -29,19 +29,23 @@
...
-
-    $logging = new LoggingClient();
-    $logger = $logging->logger($logName);
-    $logger->write($token);
+    $token = $request->request->get('token');
+    $stderr = fopen('php://stderr', 'w');
+    fwrite($stderr, $token . PHP_EOL);
+    fclose($stderr);
 
     return 'OK';
```
Comment (chingor13) [inline testapps/integration/web/index.php]: It also looks like logging_standard needs to return the name of the error log written to. I'm not sure what it should be in this case. `projects/PROJECT_ID/logs/stderr`?

Comment (tmatsuo) [inline testapps/integration/web/index.php]: seems like so, the README has an example for stderr.

**Comment by tmatsuo** [comment]
Now the tests return the logName. PTAL

**Pull request merged**